plotly/offline/offline.py

```diff
@@ -55,24 +44,20 @@ def init_notebook_mode():
         raise ImportError('`iplot` can only run inside an IPython Notebook.')
     from IPython.display import HTML, display
 
-    if not os.path.exists(PLOTLY_OFFLINE_BUNDLE):
-        raise PlotlyError('Plotly Offline source file at {source_path} '
-                          'is not found.\n'
-                          'If you have a Plotly Offline license, then try '
-                          'running plotly.offline.download_plotlyjs(url) '
-                          'with a licensed download url.\n'
-                          "Don't have a Plotly Offline license? "
-                          'Contact sales@plot.ly learn more about licensing.\n'
-                          'Questions? support@plot.ly.'
-                          .format(source_path=PLOTLY_OFFLINE_BUNDLE))
-
     global __PLOTLY_OFFLINE_INITIALIZED
     __PLOTLY_OFFLINE_INITIALIZED = True
     display(HTML('<script type="text/javascript">' +
-                 open(PLOTLY_OFFLINE_BUNDLE).read() + '</script>'))
+                 # ipython's includes `require` as a global, which
+                 # conflicts with plotly.js. so, unrequire it.
+                 'require=requirejs=define=undefined;' +
+                 '</script>' +
+                 '<script type="text/javascript">' +
+                 get_plotlyjs() +
+                 '</script>'))
```

Comment:
:( I'm not super familiar with the repercussions of this. Is this a safe thing to do?